opal/mca/accelerator/base/accelerator_base_null_component.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2017-2022 Amazon.com, Inc. or its affiliates.
+ *                         All Rights reserved.
+ * $COPYRIGHT$
+ *
+ * Additional copyrights may follow
+ *
+ * $HEADER$
+ */
+
+#ifndef MCA_ACCELERATOR_BASE_NULL_COMPONENT_H
+#define MCA_ACCELERATOR_BASE_NULL_COMPONENT_H
+
+#include "opal_config.h"
+#include "opal/mca/mca.h"
```

Comment:
In OMPI, our style is to have minimal includes in the header files, and the source files include the headers they need.  We do not use this style of a full list of headers in the main header.  Also, many of these headers look unnecessary for this component; please prune.

Reply:
pruned